### DISCUSSION TITLE
Access control on certain pages

### DISCUSSION THREAD

**Discussion by rajdevworks** [squidfunk/mkdocs-material#6872] [Q&A] [answered]
Hello, Is it possible to control access on specific pages to certain users? For example, I have few admins whom I want to give access to admin-only pages while all the users with read access should be able to access rest of the non-admin pages.

Is this something available or can be implemented?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @rajdevworks,
authentication is out of scope for a static site generator like MkDocs, definitely out of scope for the Material theme.
There were similar discussions in the past, please use the search feature:
- https://github.com/squidfunk/mkdocs-material/discussions/5875

There is also a plugin that lets you encrypt the sites, but I don't know how well it works:
- https://github.com/unverbuggt/mkdocs-encryptcontent-plugin

**Comment by ben519** [comment]
You might also consider [Scipress.io](https://www.scipress.io/) as an alternative.
